### ISSUE TITLE
zsh completions cause errors due to backticks not being escaped

### ISSUE THREAD

**Issue by chris-morgan** [crev-dev/cargo-crev#436]
Excerpt from the output of `cargo crev config completions --shell zsh`:

```zsh
_cargo__crev_commands() {
    local commands; commands=(
        "config:Local configuration" \
"crate:Crate related operations (review, verify...)" \
"id:Id (own and of other users)" \
"proof:Find a proof in the proof repo" \
"repo:Proof Repository" \
"trust:Add a Trust proof by an Id or a URL" \
"goto:Shortcut for `crate goto`" \
"open:Shortcut for `crate open`" \
"publish:Shortcut for `repo publish`" \
"review:Shortcut for `crate review`" \
"update:Shortcut for `repo update`" \
"verify:Verification Requirements" \
    )
    _describe -t commands 'cargo crev commands' commands "$@"
}
```

Those backticks aren’t escaped, and so are subshell invocations that will… well, *hopefully* fail rather than doing something weird. This mangles your prompt something awful, starting with this:

<pre><b>$ cargo crev <kbd>Tab</kbd>_cargo__crev_commands:1: command not found: crate
_cargo__crev_commands:1: command not found: crate
_cargo__crev_commands:1: command not found: repo
_cargo__crev_commands:1: command not found: crate
_cargo__crev_commands:1: command not found: repo
  cargo crev</b>
&lt;cargo crev commands completions&gt;
config                               -- Local configuration
crate                                -- Crate related operations (review, verify...)
id                                   -- Id (own and of other users)
proof                                -- Find a proof in the proof repo
publish  update  review  open  goto  -- Shortcut for
repo                                 -- Proof Repository
trust                                -- Add a Trust proof by an Id or a URL
verify                               -- Verification Requirements
</pre>

**Comment by dpc** [comment]
I think we are using `structopt` generated completion. Upgrading to recently released `clap 3.0` could fix this and improve other things.

**Comment by dpc** [comment]
Seems like `clap` `3.0` does not have completion functionality (yet?). I can't do much about it then, sorry.
